overlay/etc/chromium/master_preferences

```diff
@@ -1,26 +0,0 @@
-{
```

Comment:
Chromium pref file is not working correctly with some recent version. Their are a couple of links about the issue like this one: https://jamfnation.jamfsoftware.com/discussion.html?id=10331